### PR TITLE
libschrift: add recipe

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **libschrift/0.10.1**

libschrift is true type font rendering library, alternatvie of freetype.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by madebr — APPROVED** [review]

File: recipes/libschrift/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+set(CMAKE_C_STANDARD 99)
+set(CMAKE_C_EXTENSIONS OFF)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(KEEP_RPATHS)
+
+include_directories(source_subfolder)
+
+add_library(schrift source_subfolder/schrift.c)
```
Comment (SpaceIm) [inline recipes/libschrift/all/CMakeLists.txt:13]:
```suggestion
cmake_minimum_required(VERSION 3.8)
project(schrift LANGUAGES C)

include(conanbuildinfo.cmake)
conan_basic_setup(KEEP_RPATHS)

include(GNUInstallDirs)

add_library(schrift source_subfolder/schrift.c)
target_compile_features(schrift PRIVATE c_std_99)
set_target_properties(schrift PROPERTIES
    PUBLIC_HEADER source_subfolder/schrift.h
    WINDOWS_EXPORT_ALL_SYMBOLS ON
)

install(
  TARGETS schrift
  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
  PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
)
```

File: recipes/libschrift/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="schrift.h", dst="include", src=self._source_subfolder)
+        self.copy(pattern="*.a", dst="lib", src="lib", keep_path=False)
+        self.copy(pattern="*.so", dst="lib", src="lib", keep_path=False)
+        self.copy(pattern="*.lib", dst="lib", src="lib", keep_path=False)
+        self.copy(pattern="*.dll", dst="bin", src="bin", keep_path=False)
+        self.copy(pattern="*.dylib", dst="lib", src="lib", keep_path=False)
```
Comment (SpaceIm) [inline recipes/libschrift/all/conanfile.py:64]:
```suggestion
        cmake = self._configure_cmake()
        cmake.install()
```

**Review by SpaceIm — CHANGES_REQUESTED** [review]

**Comment by SpaceIm** [comment]
Opps, maybe restore `set(CMAKE_C_EXTENSIONS OFF)`?

**Comment by toge** [comment]
@SpaceIm 
Oh, I forget it. 
I will fix later.

**Review by SpaceIm — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Pull request merged**